arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -220,12 +219,6 @@
         "responses": {
           "202": {
             "description": "Workspace collection deleted successfully"
-          },
-          "default": {
```

Comment:
for more info see https://github.com/Azure/autorest/blob/bf30da3ae09718d336a1eca49afd7d053d83657e/docs/developer/guide/defining-clients-swagger.md#error-modeling